log4go_test.go

```diff
@@ -191,6 +193,10 @@ func TestFileLogRotationUnderFailureConditions(t *testing.T) {
 	}(LogBufferLength)
 	LogBufferLength = 0
 
+	if runtime.GOOS == "windows" {
```

Comment:
Nit: if we have platform-specific tests it seems better to break them out into a separate file for visibility.